cmd/snap/cmd_info.go

```diff
@@ -367,16 +367,17 @@ func (x *infoCmd) Execute([]string) error {
 		maybePrintID(w, both)
 		if local != nil {
 			fmt.Fprintf(w, "tracking:\t%s\n", local.TrackingChannel)
-			fmt.Fprintf(w, "refreshed:\t%s\n", local.InstallDate.Format(time.RFC3339))
 		}
-		w.Flush()
-		if local != nil {
-			fmt.Fprintf(w, "installed:\t%s\t(%s)\t%s\t%s\n", local.Version, local.Revision, strutil.SizeToStr(local.InstalledSize), notes)
+		if !(both.Updated == nil || both.Updated.IsZero()) {
+			fmt.Fprintf(w, "refreshed:\t%s\n", both.Updated.Format(time.RFC3339))
```

Comment:
not sure is sane, but "refreshed" was local only, so in theory we could use sometimes else for remote (though the origin of the value is the same)

Reply:
yep (but see my open question)